AndrewB.txt

```diff
@@ -0,0 +1,13 @@
+Hi, my name is Andrew but I go by AJ.
+
+Three words that describe me: hard working, alpha male, jackhammer, merciless, insatiable
+
+I used to have a dog named Zeke who was an 180 pound English Mastiff but he unfortunately passed away on Dec. 1st.  But I currently have a hedgehog named Fi (short for Fiona).  Hedgies are really shy so they take *months* to really warm up to you.  I've had Fi for close to 3 months now so she's starting to become okay with me.  Sometimes she naps in my lap while I listen to lectures.
```

Comment:
line 3 killed me lol.

Sorry to hear about Zeke.

My bud has a girl that is dying for a Hedgie right now.